src/smc-webapp/app_shared.cjsx

```diff
@@ -65,7 +65,7 @@ exports.NavTab = rclass
     propTypes :
         label           : rtypes.string
         label_class     : rtypes.string
-        icon            : rtypes.string
+        icon            : rtypes.oneOfType([rtypes.string,rtypes.object])
```

Comment:
Shouldn't this be `rtypes.element`?

Reply:
Probably. I need to look at the list of `rtypes` to get an idea of which ones there are. I can change that.